hw2/03-count.js

```diff
@@ -0,0 +1,11 @@
+// Add your code here
+let paragraphInfo = document.getElementById("paraInfo");
+let inputText = document.getElementById("userinput");
+inputText.addEventListener("input", (event) => {
+  const searchText = event.target.value;
+  const regex = new RegExp(searchText, "g");
+  let text = paragraphInfo.innerHTML;
+  text = text.replace(/(<mark style="background-color: yellow;">|<\/mark>)/gim, "");
```

Comment:
Do not use the mark tag, instead just rely on CSS for styling. 

Do not use in-line CSS, just add a class and define the styles in the CSS file.